nearcache_map.go

```diff
@@ -360,6 +361,20 @@ func (ncm *nearCacheMap) Remove(ctx context.Context, m *Map, key interface{}) (i
 	return m.removeFromRemote(ctx, key)
 }
 
+func (ncm *nearCacheMap) RemoveAll(ctx context.Context, m *Map, predicate predicate.Predicate) (err error) {
+	keys, err := m.GetKeySetWithPredicate(ctx, predicate)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		for _, key := range keys {
+			key, err = ncm.toNearCacheKey(key)
+			ncm.nc.Invalidate(key)
+		}
+	}()
```

Comment:
This code doesn't look correct, all needed should be something like this:
```
defer func() {
    ncm.Clear(ctx, m)
}()
```

Reply:
Fixed at https://github.com/hazelcast/hazelcast-go-client/pull/873/commits/59d4abf8629e69964b3905af346d81a7dbfb6fd3.